### PR TITLE
[ML] Fix log spam and disable ILM/SLM history for native ML tests

### PR DESCRIPTION
**Pull request by benwtrent**
This adds a dependency to ingest common. This removes the log spam resulting from basic plugins being enabled that require the common ingest processors.

### CODE REVIEW

File: x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeIntegTestCase.java [hunk 1]
```diff
@@ -191,10 +190,6 @@ protected boolean upgradeMode() {
         return client().execute(PutFilterAction.INSTANCE, new PutFilterAction.Request(filter)).actionGet();
     }
 
-    protected GetFiltersAction.Response getMlFilters() {
-        return client().execute(GetFiltersAction.INSTANCE, new GetFiltersAction.Request()).actionGet();
-    }
```
Comment (droberts195) [inline x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeIntegTestCase.java:196]: Why does this method have to be removed as part of this PR?

Comment (benwtrent) [reply x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeIntegTestCase.java:196]: It doesn't have to be. But it was unused and I was exploring this class for fixing this problem. 

Following the [boy scout rule](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)

**Review by droberts195 — APPROVED** [review]
LGTM

**Review by droberts195 — APPROVED** [review]
LGTM

**Pull request merged**